libs/tup-components/src/projects/details/ProjectDetails.tsx

```diff
@@ -130,6 +131,30 @@ const Grant: React.FC<{
   );
 };
 
+const NewAllocation = () => (
+  <>
+    <a href="https://submit-tacc.xras.org/" target="_blank" rel="noreferrer">
+      <Button className={styles['tap-button']} type="link">
```

Comment:
The `<Button>` used was `type="link"`, so it would have looked like a link... just use a link 🙂.

In other words, rather than an `<a>` wrapping a `<button>`[^1], which is not permitted in HTML[^2], just have the `<a>` tag, **no** `<button>`.

[^1]: [A `<Button>` (React) creates a `<button>` (HTML).](https://github.com/TACC/tup-ui/blob/9402505/libs/core-components/src/lib/Button/Button.tsx#L76)
[^2]: [An `<a>` permits most elements inside](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#technical_summary), but **not** interactive content (e.g. a button): "[…] except that no descendant may be [interactive content](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#interactive_content)".

Reply:
We need a general component for the case of "link that displays as a button" since it's a pattern that comes up multiple times in the design. I say we defer this (putting a button inside an anchor tag does *work*) and open a new ticket for button-style links.

Reply:
[FP-1971](https://jira.tacc.utexas.edu/browse/FP-1971): Link That Can Display as Button

I see the goal is a link that looks like a button. The `<Button>` tag still seems inappropriate. There is yet another (and simpler) alternative[^1], but I see benefits to both short-term solutions.

[^1]: `<a href="https://submit-tacc.xras.org/" target="_blank" rel="noreferrer" class="c-button c-button--secondary">`